### DISCUSSION TITLE
All the maps and skins are seemingly missing from the client

### DISCUSSION THREAD

**Discussion by DecayingEagle** [ppy/osu#25585] [Q&A] [answered]
### Type

Other

### Type

Other
### Bug description

I opened my game, there were no errors in notifications or anything. I made no changes to the `osu` folder nor `files` folder, but i did check the size of the files folder and it seems that the files are still there. I also took a quick peek at some files and from the short peek I didn't notice any corruption. 

I also tried downloading a map and making a map, and that worked but I did not fix the issue.

As per usual if there is any more info needed I'll supply it asap.

### Specs
OS: Arch Linux x86_64
Kernel: 6.6.2-arch1-1
CPU: Intel i5-9400F (6) @ 4.100GHz
GPU: NVIDIA GeForce RTX 2060 Rev. A
Memory: 11289MiB / 15901MiB

### Version

2023.1141.1

### Logs

[database.log](https://github.com/ppy/osu/files/13466385/database.log)
[input.log](https://github.com/ppy/osu/files/13466386/input.log)
[network.log](https://github.com/ppy/osu/files/13466389/network.log)
[performance.log](https://github.com/ppy/osu/files/13466390/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13466391/runtime.log)

**Comment by peppy** [comment]
Please show us a screenshot of your osu! folder, especially any `.realm` files.

**Reply by DecayingEagle** [reply]
![image](https://github.com/ppy/osu/assets/46646694/f07f19e8-5961-4e73-8fd1-00962307fe5d)

Oh yeah, there is `client_1700869215_corrupt.realm` I missed that, mb

**Reply by peppy** [reply, marked as answer]
Could you please try renaming `client.realm` to something else temporarily, and copying the `_corrupt.realm` file in its place? Please keep a copy of the "corrupt" one in case we need it for analysis.

**Reply by DecayingEagle** [reply]
Seems like that fixed the issue, thanks for help.
